src/main/java/com/cascadebot/cascadebot/commands/developer/GuildCommand.java

```diff
@@ -0,0 +1,38 @@
+package com.cascadebot.cascadebot.commands.developer;
+
+import com.cascadebot.cascadebot.commandmeta.CommandContext;
+import com.cascadebot.cascadebot.commandmeta.ICommandExecutable;
+import com.cascadebot.cascadebot.commandmeta.ICommandRestricted;
+import com.cascadebot.cascadebot.commandmeta.Module;
+import com.cascadebot.shared.SecurityLevel;
+import net.dv8tion.jda.core.entities.Member;
+
+import java.util.Set;
+
+public class GuildCommand implements ICommandRestricted {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
```

Comment:
We should reply with **something** here. maybe for now just reply with `Do [prefix]guild save to save the guild data`
@binaryoverload at some point we should add automatic formatting to like context.reply()

Reply:
What do you mean? @weeryan17